.templates/transmission/volumes.yml

```diff
@@ -0,0 +1,21 @@
+
+  transm_download_volume:
+    driver: local
+    driver_opts:
+       o: bind
+       type: none
+       device: <transm_download_volume absolute path on disk>
```

Comment:
For now, can you make these actual paths?
```
/home/${USER}/IOTstack/volumes/transmission/downloads
```
Or something.

Otherwise it causes an error when starting:
![image](https://user-images.githubusercontent.com/6442613/79955025-117fad00-8433-11ea-9737-bda306b35ae6.png)

Reply:
Sure! I put it as a bookmark to be changed after running menu.sh, so anyone can use the absolute path he/she want (e.g. I have them on an external hd).